src/plugins/spdk/spdk_env/memory.c

```
@@ -0,0 +1,80 @@
+	return false;
+}
+
+int32_t spdk_mem_get_numa_id(const void *buf, uint64_t *size)
+{
+	if (size) {
+		*size = UINT64_MAX;
+	}
+	return SPDK_ENV_NUMA_ID_ANY;
+}
+
+int spdk_mem_get_fd_and_offset(void *vaddr, uint64_t *offset)
```

Cppcheck (2.21.0)

[style] The function 'spdk_mem_get_fd_and_offset' is never used.

(unusedFunction)